src/openrct2/config/Config.h

```diff
@@ -214,11 +215,11 @@ enum class TemperatureUnit : int32_t
     Fahrenheit
 };
 
-enum SCALE_QUALITY
+enum class ScaleQuality : int32_t
 {
-    SCALE_QUALITY_NN,
-    SCALE_QUALITY_LINEAR,
-    SCALE_QUALITY_SMOOTH_NN
+    NN, //Nearest Neighbour
```

Comment:
Comments should be on the line before, not on the same line as code.

Any reason this couldn't just be NearestNeighbour instead of NN? A bit long, but a lot clearer.

Reply:
You're right, that's a way better idea, was just coping the previous enum without thinking